### ISSUE TITLE
Commented out code.

### ISSUE THREAD

**Issue by AndreyBocharnikov** [SayedNadim/Global-and-Local-Attention-Based-Free-Form-Image-Inpainting#18]
Please make it clear why [the code at](https://github.com/SayedNadim/Global-and-Local-Attention-Based-Free-Form-Image-Inpainting/blob/bae972d239e2753de588c4f48e2a6d17c42e5d1b/model/network.py#L214) `# self.linear = nn.Linear(self.cnum * 4 * 8 * 8, 1)` is commented out? In the paper, you said " A dropout layer is incorporated in the discriminator to allow stochasticity
in the model", but in code, both the Linear layer and the dropout are commented out, were they also commented out during training or they were used as you said in the paper? Thank you.

**Comment by SayedNadim** [comment]
Hi, Sorry for the late reply. In our final version, we have used the dropout layer for training, as per the paper. I may have commented it out to check something, but unfortunately, I cannot remember why. Hope this helps.
Cheers!
